ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTest.cs

```diff
@@ -0,0 +1,44 @@
+using LeetCodeTraining.TwoSum1;
+namespace LeetCodeTraining.Tests
+{
+    public class TwoSumSolutionTest
+    {
+        [Fact]
+        public void TwoSumArrayWith2Nums()
+        {
+            //Arrange
+            int[] nums = new int[] { 2,7,11,15 };
```

Comment:
Please rename variables as for first test group - num1, the second nums2, the third - nums3 to be more clear.
Also please add extra lines for better reading the code between groups like this:
![image](https://github.com/ArinaSiselina/ScnSoft-Education/assets/101803780/371248a3-2eae-4e47-a2f5-3a51de43161b)